package de.marvinzichner.bruchrechner.exception;

public class ExitException extends Exception {


  public ExitException() {
    super("Exit Called");
  }
}
